persistence/src/main/java/org/dswarm/persistence/monitoring/EntityIdentification.java

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (C) 2013 – 2015 SLUB Dresden & Avantgarde Labs GmbH (<code@dswarm.org>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.dswarm.persistence.monitoring;
+
+import java.util.Arrays;
+
+import com.google.common.base.CharMatcher;
+import com.google.common.base.Joiner;
+import com.google.common.base.Splitter;
+import com.google.common.base.Strings;
+import org.apache.commons.lang3.StringUtils;
+import org.slf4j.MDC;
+
+import org.dswarm.persistence.model.ExtendedBasicDMPJPAObject;
+import org.dswarm.persistence.service.UUIDService;
+
+final class EntityIdentification {
+
+	private static final CharMatcher MATCHER =
+			CharMatcher.ASCII
+					.and(CharMatcher.JAVA_LETTER_OR_DIGIT)
+					.negate()
+					.or(CharMatcher.WHITESPACE);
+
+	private final String className;
+	private final String description;
+	private final String name;
+	private final String uuid;
+
+	private EntityIdentification(final String className, final String uuid, final String name, final String description) {
+		this.className = className;
+		this.uuid = uuid;
+		this.name = name;
+		this.description = description;
```

Comment:
Could be. The name's truncated though, so the description might be cut off anyway. Should I remove it entirely?


> On Mar 27, 2015, at 10:17 AM, Bo Ferri <notifications@github.com> wrote:
> 
> In persistence/src/main/java/org/dswarm/persistence/monitoring/EntityIdentification.java:
> 
> > +	private static final CharMatcher MATCHER =
> > +			CharMatcher.ASCII
> > +					.and(CharMatcher.JAVA_LETTER_OR_DIGIT)
> > +					.negate()
> > +					.or(CharMatcher.WHITESPACE);
> > +
> > +	private final String className;
> > +	private final String description;
> > +	private final String name;
> > +	private final String uuid;
> > +
> > +	private EntityIdentification(final String className, final String uuid, final String name, final String description) {
> > +		this.className = className;
> > +		this.uuid = uuid;
> > +		this.name = name;
> > +		this.description = description;
> maybe description is too much here, or?
> 
> —
> Reply to this email directly or view it on GitHub.
>